src/core/client/stream/tabs/Profile/Settings/IgnoreUserSettingsContainer.css

```diff
@@ -1,7 +1,3 @@
-.root {
-  max-width: 440px;
```

Comment:
How does this behave if you up the internal browser default text size? Do we get excessive line-wrapping that makes the interface un-usable?